extensions/wikia/CreateNewWiki/js/WikiBuilder.js

```diff
@@ -84,7 +86,9 @@ define('ext.createNewWiki.builder', ['ext.createNewWiki.helper', 'wikia.tracker'
 		wikiDomainLabel = $nameWikiWrapper.find('label[for=wiki-domain]');
 		wikiDomainError = $nameWikiWrapper.find('.wiki-domain-error');
 		wikiDomainCountry = $nameWikiWrapper.find('.domain-country');
-		wikiLanguage = $nameWikiWrapper.find('select[name=wiki-language]');
+		nameWikiSubmitError = $nameWikiWrapper.find('.submit-error');
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "nameWikiSubmitError" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)